lib/src/SwipeTo.dart

```diff
@@ -163,14 +173,14 @@ class _SwipeToState extends State<SwipeTo> with SingleTickerProviderStateMixin {
                   opacity: _leftIconAnimation.value,
                   duration: widget.animationDuration,
                   curve: Curves.decelerate,
-                  child: widget.rightSwipeWidget != null
-                      ? widget.rightSwipeWidget
-                      : Icon(
-                          widget.iconOnRightSwipe,
-                          size: widget.iconSize,
-                          color: widget.iconColor ??
-                              Theme.of(context).iconTheme.color,
-                        ),
+                  child: widget.rightSwipeWidget ??
+                      // TODO: Replace with default values once we remove the deprecated properties
```

Comment:
@AyushBherwani1998  I guess this will be the default `Icon` widget with specific icon to be displayed underneath. Right?

Reply:
Yes, this will ensure the existing users don't go through a breaking change. Developers who are using `iconOnRightSwipe` and related properties, they will get a warning in the analyzer saying they should use the `rightSwipeWidget` and `leftSwipeWidget`.